modules/server-node/prisma-sqlite/schema.prisma

```diff
@@ -208,19 +239,15 @@ model OnchainTransaction {
   logs              String?
   cumulativeGasUsed String?
   byzantium         Boolean?
-
-  // Channel fields
-  status         String  @default("submitted") // no enums
-  reason         String // no enums
-  error          String?
-  channelAddress String
-  channel        Channel @relation(fields: [channelAddress], references: [channelAddress])
+  status            Int?
 
   createdAt DateTime @default(now())
 
-  @@unique([transactionHash, chainId])
-  @@map(name: "onchain_transaction")
-  Transfer Transfer[] @relation("Transaction")
+  transfer Transfer? @relation
+
+  onchainTransaction OnchainTransaction? @relation()
```

Comment:
Remove optional `?`

Also, since f-key will be here, should be: `@relation(name: "OnchainTransaction", fields: [onchainTransactionId], references: [id])`

Also add a `onchainTransactionId` f-key field ^